mypy/types.py

```diff
@@ -253,9 +395,10 @@ class CallableType(FunctionLike):
     arg_types = None  # type: List[Type]  # Types of function arguments
     arg_kinds = None  # type: List[int]   # mypy.nodes.ARG_ constants
     arg_names = None  # type: List[str]   # None if not a keyword argument
-    min_args = 0                    # Minimum number of arguments
-    is_var_arg = False              # Is it a varargs function?
-    ret_type = None  # type:Type    # Return value type
+    min_args = 0                    # Minimum number of arguments; derived from arg_kinds
+    is_var_arg = False              # Is it a varargs function?  derived from arg_kinds
+    ret_type = None  # type: Type   # Return value type
+    fallback = None  # type: Instance
```

Comment:
Repeated from base class `FunctionLike`. I wonder why mypy doesn't complain about this.